xml/templates/civicrm_state_province.tpl

```diff
@@ -4090,3 +4090,34 @@ INSERT INTO civicrm_state_province (country_id, abbreviation, name) VALUES
 (1016, "14", "Al Janūbīyah"),
 (1016, "16", "Al Wusţá"),
 (1016, "17", "Ash Shamālīyah");
+
+-- Add missing provinces for Moldova
+(NULL, 1142, 'AN', 'Anenii Noi'),
```

Comment:
Thanks @jsuchogorska There's just some technical issues:
* You'll need to remove the NULLs from these since there should only be 3 values on each row.
* You'll need to change the semicolon on line 4092 to a comma.
* You'll need to remove the comma from line 4120 below.

Reply:
@demeritcowboy thank you. Fixed.